from_mwt_ds/DataScience/estimate.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+from estimators import bandits
+from estimators.bandits import ips
+
+def estimate_bucket(bucket: pd.DataFrame, config: dict) -> dict:
+    if len(bucket) == 0 or config == {}:
+        return {}
+    else:
+        e = bandits.ips.Estimator()
```

Comment:
Although we create estimator object here:
- we are still creating ips estimator always (no matter what is specified in the config)
- we are not using it anywhere else in this function (which seems wrong)